spec/requests/stories/feeds_spec.rb

```diff
@@ -0,0 +1,37 @@
+require "rails_helper"
+
+RSpec.describe "Stories::FeedsIndex", type: :request do
+  let(:title) { "My post" }
+  let!(:article) { create(:article, title: title, featured: true) }
+
+  describe "GET feeds index" do
+    let(:response_json) { JSON.parse(response.body) }
```

Comment:
you don't really need this, you can use `response.parsed_body` which already returns the parsed JSON